moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -249,18 +259,25 @@ const ComponentForm = ({
                 id="completion-date"
                 name="completionDate"
                 control={control}
-                render={({ onChange, value, ref }) => (
-                  <DatePicker
-                    inputRef={ref}
-                    value={value}
-                    onChange={onChange}
-                    clearable={true}
-                    emptyLabel="mm/dd/yyyy"
-                    format="MM/dd/yyyy"
-                    variant="outlined"
-                    label={"Completion date"}
-                  />
-                )}
+                render={({ onChange, value, ref }) => {
+                  return (
+                    <DatePicker
+                      inputRef={ref}
+                      value={parseDate(value)}
+                      onChange={(date) => {
+                        const newDate = date
+                          ? format(date, "yyyy-MM-dd OOOO")
```

Comment:
i added the timezone because `new  Date()` was assuming it was midnight GMT which meant the calendar day showing up was the day before (it was converting it to CDT)